extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -595,9 +595,9 @@ export class SqlDatabaseTree {
 
 	public selectedDbs(): string[] {
 		let result: string[] = [];
-		this._databaseTable.dataValues?.forEach((arr) => {
+		this._databaseTable.dataValues?.forEach((arr, index) => {
 			if (arr[0].value === true) {
-				result.push(arr[1].value.toString());
+				result.push(this._dbNames[index]);
```

Comment:
Why are you changing this? I typically don't think it's a good idea to have things tied together like this that aren't actually directly related at all. Shouldn't the row have the name in it somewhere?

Reply:
So now instead of text values, I am using flex container inside the cells. 
![image](https://user-images.githubusercontent.com/6816294/112561871-4ad34200-8d93-11eb-928f-84edc2732626.png)

Over here it is not possible to retrieve text in the previous way.

Reply:
Probably, I can cast the general component to flex container and get the value of the second component in that container.